packages/admin/dashboard/src/routes/orders/order-create-exchange/components/exchange-create-form/exchange-outbound-section.tsx

```diff
@@ -62,6 +63,13 @@ export const ExchangeOutboundSection = ({
     fields: "*prices,+service_zone.fulfillment_set.location.id",
   })
 
+  const outboundShippingOptions = shipping_options.filter(
+    (shippingOption) =>
+      !!shippingOption.rules.find(
+        (r) => r.attribute === "is_return" && r.value === "false"
```

Comment:
question: Would it make sense to allow a `rules` filter in the API Route, so we can filter on the backend?

Reply:
You mean like `is_return` flag that we have on the store endpoint? or how would the query look like?

Reply:
Hmm, forget what I said, I don't think we can easily create a `rules` filter for the endpoint.

Basically, what I had in mind was something like:
```
GET /admin/shipping-options?rules[attribute]=is_return&rules[value]=false
```

Reply:
Yeah, also issue is if we want to filter by `enabled_in_store` as well in the same query. I can open a ticket to revisit filtering options on the admin endpoint.